### ISSUE TITLE
Addresses getting out of sync

### ISSUE THREAD

**Issue by derula** [RyogaMasaki/jstrings#3]
I have 2 binary files, one Japanese, one English.

In both files, it finds the same English string. `[Save Error]`.

In the Japanese binary, it claims the string is at offset `0xf85cb6`, in the English binary it supposedly finds it at `0xf85cb2`.

The actual position of the string in both binaries is `0xf85cec`.

Somehow, the address is drifting while processing the file.

I will try to reproduce with a smaller example. If I can't, I can link to the original binaries I'm having trouble with.

**Comment by drojaazu** [comment]
I *thought* I had noticed incorrect addresses at one point. Thanks, I will look into this.

**Comment by derula** [comment]
I tried to narrow it down. While I failed to create a smaller example, I noticed that the first time the addresses shifted (at least with my example) was somewhere between 0xf1e29 and 0x10767c - 0x100000 perhaps? The string at 0xf1e29 had the correct address, and the one at 0x10767c is wrongly reported as 0x10767a.

I copied the block between those addresses into a new file, and both addresses were correct.
